.github/workflows/sonarcloud.yml

```
@@ -1,86 +1,77 @@
-            if [ -d "$dir/build/classes/java/test" ]; then TEST_BINARIES="$TEST_BINARIES,$dir/build/classes/java/test"; fi
-            if [ -f "$dir/build/reports/jacoco/test/jacocoTestReport.xml" ]; then JACOCO_REPORTS="$JACOCO_REPORTS,$dir/build/reports/jacoco/test/jacocoTestReport.xml"; fi
-          done
-
-          # Remove leading commas
-          echo "sources=${SOURCES#,}" >> $GITHUB_OUTPUT
-          echo "tests=${TESTS#,}" >> $GITHUB_OUTPUT
-          echo "binaries=${BINARIES#,}" >> $GITHUB_OUTPUT
-          echo "test_binaries=${TEST_BINARIES#,}" >> $GITHUB_OUTPUT
-          echo "jacoco_reports=${JACOCO_REPORTS#,}" >> $GITHUB_OUTPUT
+      - name: Build, Run Tests and Generate JaCoCo Reports
+        working-directory: ./project/backend 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)